### PR TITLE
LRDOCS-6454

### PR DESCRIPTION
**Pull request by Preston-Crary**

### CODE REVIEW

File: portal-kernel/src/com/liferay/portal/kernel/transaction/Isolation.java [hunk 1]
```diff
@@ -19,7 +19,7 @@
 import java.util.Map;
 
 /**
- * Defines how transations are visible to those outside the transaction.
+ * Defines the isolation level from other transactions.
```
Comment (jhinkey) [inline portal-kernel/src/com/liferay/portal/kernel/transaction/Isolation.java]: As an example, would you describe the READ_COMMITTED level?

Do you mean _this_ transaction is forbidden to do dirty reads?

Comment (Preston-Crary) [reply portal-kernel/src/com/liferay/portal/kernel/transaction/Isolation.java]: It's declarative. The transaction will not see any dirty data.

Comment (Preston-Crary) [reply portal-kernel/src/com/liferay/portal/kernel/transaction/Isolation.java]: You cannot try to do a dirty read, you can permit to see them if they are there.

**Comment by jhinkey** [comment]
Started reviewing

**Comment by jhinkey** [comment]
sent https://github.com/Preston-Crary/liferay-portal/pull/732

**Pull request closed without merging**